src/test/ebpf_yaml.cpp

```
@@ -87,7 +87,9 @@ static EbpfMapDescriptor test_map_descriptor = {.original_fd = 0,
                                                 .max_entries = 4,
                                                 .inner_map_fd = 0};
 
-static const EbpfMapDescriptor& ebpf_get_map_descriptor(int, const ProgramInfo&) { return test_map_descriptor; }
+static const EbpfMapDescriptor& ebpf_get_map_descriptor(int, const std::vector<EbpfMapDescriptor>&) {
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)